### DISCUSSION TITLE
Question about Async backups/replication

### DISCUSSION THREAD

**Discussion by srisch** [seaweedfs/seaweedfs#5124] [Q&A] [answered]
Hello!

We're currently using seaweed in production and are loving it!  I'm working on backups to AWS S3 for a seaweed s3 bucket called "long-term-storage" and i'm trying to figure out the best way to do it and I had a few questions.

- Is there a way to currently continually backup a specific bucket to s3? It wasn't clear from the current docs ( Unless i missed it ) - Not looking for incremental changes, just when a new file is added it gets copied to AWS s3.
- Are the volumes backed up as a whole or are individual files backed up to the s3 sink? 
- Are the files still encrypted by the Filer, or will I be able to access the raw files on s3?

Thank you!

**Comment by chrislusf** [comment, marked as answer]
1. `weed filer.backup`
2. individual files. Volumes backup is done using `weed backup`
3. As raw files on s3.

**Reply by srisch** [reply]
Awesome thank you so much!
